@scale/vagrant_libvirt/README.md

```diff
@@ -0,0 +1,144 @@
+# Vagrant and Libvirt plugin for @scale deployment
+
+## Introduction
+* "Vagrant - the command line utility for managing the life-cycle of virtual machines." (version: 1.9.2)
+* Vagrant-libvirt plugin add a provider to Vagrant which allowing Vagrant to control and provision machines via libvirt toolkit. (version 1.3.1)
+* Vagrant up, destroy, suspend, resume, halt, ssh, reload, package and provision commands.
+
+# Setup and Execution
+
+## Vagrant and libvirt setup
+Here we select a VM with Ubuntu 16.04 installed as Vagrant host server.
+* Vagrant host Environment Preparation: 
+   
+1.  Install Vagrant dependency
+
+	$ sudo apt-get install ruby-full
+
+	$ sudo apt install ruby-bundler
+
+	$ git clone https://github.com/mitchellh/vagrant.git
+
+	$ cd vagrant
+
+	$ bundle install
+
+2.  Install latest version vagrant from https://www.vagrantup.com/downloads.html
+    
+	$ wget https://releases.hashicorp.com/vagrant/1.9.2/vagrant_1.9.2_x86_64.deb 
+
+	$ sudo dpkg -i vagrant_1.9.2_x86_64.deb
+
+	$ vagrant --version  **"to make sure Vagrant version as your expectation"**
+
+3.  Need to make sure your have all the build dependencies (in Ubuntu) installed for vagrant-libvirt. 
+
+	**Uncomment all "deb-src" sources in "/etc/apt/sources.list"**
+
+	$ sudo apt-get update
+
+	$ sudo apt-get build-dep vagrant ruby-libvirt
+
+	$ sudo apt-get install qemu libvirt-bin ebtables dnsmasq
+
+	$ sudo apt-get install libxslt-dev libxml2-dev libvirt-dev zlib1g-dev ruby-dev
+
+	$ sudo adduser $USER libvirtd
+
+4.  Check the qemu version is not "infrasim-qemu".
+	
+	$ qemu-system-x86_64 --version
+
+	- If the version is "QEMU emulator version infrasim-qemu_2.6.2-1.0.19ubuntu16.04, Copyright (c) 2003-2008 Fabrice Bellard",
+	  run command first: **$ sudo dpkg -r infrasim-qemu**
+
+	- If the version is "QEMU emulator version 2.5.0 (Debian 1:2.5+dfsg-5ubuntu10.9), Copyright (c) 2003-2008 Fabrice Bellard", 
+	  go to install vagrant-libvirt plugin.
+		
+
+5.  Need to install vagrant-libvirt use standard vagrant plugin installation method.
+    
+	$ vagrant plugin install vagrant-libvirt
+
+	$ vagrant plugin list
+
+        vagrant-libvirt (0.0.37)
+
+## Use "Packer" and Packer templates to build vagrant box.
+
+   Packer build templates found in [ chef/bento github repo.](https://github.com/chef/bento).
+    
+	$ git clone https://github.com/chef/bento
+
+	$ cd bento
+
+	$ sudo packer build -only qemu -var "headless=true" ubuntu-14.04-amd64.json
+
+	$ vagrant box add builds/ubuntu-14.04.libvirt.box --name "ubuntu1404"
```

Comment:
It fails on execution of the command "vagrant box add builds/ubuntu-14.04.libvirt.box --name" at my side, not sure if the "builds/ubuntu-14.04.libvirt.box" meant for directory path to file or url? If directory to file, I would recommend to change it to dir/to/ubuntu-14.04.libvirt.box?